drag_and_drop_v2/translations/en/LC_MESSAGES/text.po

```diff
@@ -0,0 +1,168 @@
+msgid "The Top Zone"
+msgstr ""
+
+#: default_data.py
+msgid "The Middle Zone"
+msgstr ""
+
+#: default_data.py
+msgid "The Bottom Zone"
+msgstr ""
+
+#: default_data.py
+msgid "Use this zone to associate an item with the top layer of the triangle."
+msgstr ""
+
+#: default_data.py
+msgid ""
+"Use this zone to associate an item with the middle layer of the triangle."
+msgstr ""
+
+#: default_data.py
+msgid ""
+"Use this zone to associate an item with the bottom layer of the triangle."
+msgstr ""
+
+#: default_data.py
+#, python-brace-format
+msgid "Correct! This one belongs to {zone}."
+msgstr ""
+
+#: default_data.py
+msgid "No, this item does not belong here. Try again."
+msgstr ""
+
+#: default_data.py
+msgid "You silly, there are no zones for this one."
+msgstr ""
+
+#: default_data.py
+msgid "Drag the items onto the image above."
+msgstr ""
+
+#: default_data.py
+msgid "Good work! You have completed this drag and drop problem."
+msgstr ""
+
+#: default_data.py
+msgid "Goes to the top"
+msgstr ""
+
+#: default_data.py
+msgid "Goes to the middle"
+msgstr ""
+
+#: default_data.py
+msgid "Goes to the bottom"
+msgstr ""
+
+#: default_data.py
+msgid "I don't belong anywhere"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Title"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid ""
+"The title of the drag and drop problem. The title is displayed to learners."
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Drag and Drop"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Show title"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Display the title to the learner?"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Problem text"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "The description of the problem or instructions shown to the learner"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Show \"Problem\" heading"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Display the heading \"Problem\" above the problem text?"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Weight"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "The maximum score the learner can receive for the problem"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Item background color"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "The background color of draggable items in the problem."
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Item text color"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Text color to use for draggable items."
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Problem data"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid ""
+"Information about zones, items, feedback, and background image for this "
+"problem. This information is derived from the input that a course author "
+"provides via the interactive editor when configuring the problem."
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid ""
+"Information about current positions of items that a learner has dropped on "
+"the target image."
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Indicates whether a learner has completed the problem at least once"
+msgstr ""
```

Comment:
@asadiqbal08 

> I am not considering .html files in this process because we cannot refer the ugettext runtime service in templates.

Strings that are marked for translation in `js_templates.html` are passed to `gettext` on the client via [this Handlebars helper](https://github.com/edx-solutions/xblock-drag-and-drop-v2/blob/master/drag_and_drop_v2/public/js/drag_and_drop_edit.js#L9), so if translations are available for them, I *think* they should get picked up dynamically.

But if there is currently no method for automatically extracting strings from all relevant file types<sup>1</sup>, it might make sense to address my comment about `js_templates.html` at another time.

CC'ing @bradenmacdonald to see what he thinks.

<sup>1</sup>That's what I'm getting from your [comment](https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/66#issuecomment-203799017) below.